web/src/components/ConsentBanner.js

```diff
@@ -1,8 +1,8 @@
 import { Button } from '@cmsgov/design-system-core';
 import PropTypes from 'prop-types';
-import React, { Fragment, PureComponent } from 'react';
+import React, { Fragment, useState } from 'react';
 
-let cookie = name => {
+const cookie = name => {
```

Comment:
The old way would read the cookies when the module was loaded, parse them, and cache that for eternity. That made the testing more difficult - we need to unload and reload the module for each test. It was meant to be an efficiency thing, but this cookie parsing will only happen on the login page, so it doesn't really seem worth optimizing. I took out this cacheing so it'd be easier to read and also easier to test.